Check if CADD plugin arguments are Tabix-indexed files

nuno-agostinho opened the pull request:
Added safeguards to check if files are valid, similar to #458.

Also removed comment stating that CADD files were only available for GRCh37.

diegomscoelho commented on CADD.pm:
```diff
@@ -71,6 +71,11 @@ sub new {
 
+  # Check files in arguments
+  my @params = @{$self->params};
+  die "ERROR: No CADD files specified\n" unless @params > 0;
```
```suggestion
  die "\nERROR: No CADD files specified\nTip: Add a file after comma, example:\nvep ... --custom CADD,/FULL_PATH_TO_CADD_FILE/whole_genome_SNVs.tsv.gz\n" unless @params > 0;
```

diegomscoelho reviewed (commented):
This warning will not check if it's a valid file. It will check if it exists anything after `--custom CADD,`

If you run: `vep --plugin CADD,blabla`

It's going to pass this test and fall into other check: `ERROR: Data file blabla not found`

nuno-agostinho commented:
Hey @diegomscoelho, this PR prevents the CADD plugin from running by throwing an error in the following conditions:

* If no plugin parameters are specified (based on `@params > 0`)
* If any plugin parameter is not an existing file (by passing all arguments to [add_file][])
* If any given file has no respective and existing Tabix-indexed file (by passing all arguments to [add_file][])

[add_file]: https://github.com/Ensembl/ensembl-variation/blob/ac8178e987511a0bd2357334150c4271e6698896/modules/Bio/EnsEMBL/Variation/Utils/BaseVepTabixPlugin.pm#L143-L162

I fixed the PR title to clarify this and changed the error message to be more akin to your suggestion. Thanks!

nuno-agostinho commented:
Superseded by #470
